CHARTER.md

```diff
@@ -0,0 +1,137 @@
+resignation or by consensus with at least five voting members to be present. A vote in a TSC meeting can be used to change a regular TSC member to a voting
+TSC member, or to change a voting TSC member to a regular TSC member.
+
+No more than one-fourth of the TSC voting members may be affiliated with the same employer. If a change in TSC voting
+membership or a change of employment by a TSC voting member creates a situation where more than one-fourth of the TSC
+voting membership shares an employer, then the situation must be immediately remedied by the removal of voting member
+status from one or more TSC voting members affiliated with the over-represented employer(s).
+
+The TSC shall meet regularly using tools that enable participation by the community (e.g. weekly on a Google Hangout On
+Air, or through any other appropriate means selected by the TSC). The meeting shall be directed by the TSC Chairperson.
+Responsibility for directing individual meetings may be delegated by the TSC Chairperson to any other TSC voting member.
+Minutes or an appropriate recording shall be taken and made available to the community through accessible public
+postings.
+
+TSC voting members are expected to regularly participate in TSC meetings and issue triaging. Meeting notes will be released such that absent members of the TSC can provide feedback in any way.
+
+A TSC voting member is automatically converted to a TSC regular member if they do not participate in three consecutive
+TSC votes.
+
+## Section 4. Roles & Responsibilities of the TSC
+
+Subject to such policies as may be set by the CPC, the TSC voting members are responsible for all technical development
+within the webpack project, including:
+
+* Setting release dates.
+* Release quality standards.
+* Technical direction.
+* Project governance and process.
+* GitHub repository management.
+* Conduct and maintain guidelines defined by the OpenJS Foundation Cross Project Council.
+* Maintaining the list of additional collaborators.
+* Development process and any coding standards.
+* Mediating technical conflicts between collaborators or `webpack` projects.
+
+The TSC voting members will define webpack project's release vehicles.
+
+### Section 4.1. webpack Project Operations
+
+The TSC voting members will establish and maintain a development process for the webpack project. The development
+process will establish guidelines for how the developers and community will operate. It will, for example, establish
+appropriate timelines for TSC review (e.g. agenda items must be published at least a certain number of hours in advance
+of a TSC meeting).
+
+Note that project operations remain subject to any relevant policy or process specified by the OpenJS Foundation board or Cross Project Council.
+
+### Section 4.2. Decision-making, Voting, and/or Elections
+
+#### Section 4.2.1. Elections
+
+These processes are described in webpack's [GOVERNANCE](GOVERNANCE.md) document.
```

Comment:
It seems that the election process is actually described in section 3 above, no?

Maybe move it here?

Reply:
I don't see anything related to elections on section 3 🤔

Reply:
Well, this is where you describe how TSC members are chosen, so that’s the closest you have to elections.

Reply:
Let me see if I understood this correctly: you're asking for us to move this election sub-section to be part of section 3?

(The charter template on the CPC's repo mentions elections as part of section 4, that's why I added here) -- because there's no reference in section 3 of how someone becomes a TSC member.